video/out/opengl/context_drm_egl.c

```diff
@@ -527,12 +530,15 @@ static void queue_flip(struct ra_ctx *ctx, struct gbm_frame *frame)
     data->log = ctx->log;
 
     if (atomic_ctx) {
+        if(ctx->vo->opts->drm_opts->drm_send_hdr_meta)
+            drm_send_hdr_metadata(atomic_ctx, &p->out_color, &p->hdr_blob_id);
+
         drm_object_set_property(atomic_ctx->request, atomic_ctx->draw_plane, "FB_ID", p->fb->id);
         drm_object_set_property(atomic_ctx->request, atomic_ctx->draw_plane, "CRTC_ID", atomic_ctx->crtc->id);
         drm_object_set_property(atomic_ctx->request, atomic_ctx->draw_plane, "ZPOS", 1);
 
         ret = drmModeAtomicCommit(p->kms->fd, atomic_ctx->request,
-                                  DRM_MODE_ATOMIC_NONBLOCK | DRM_MODE_PAGE_FLIP_EVENT, data);
+                                  DRM_MODE_ATOMIC_ALLOW_MODESET | DRM_MODE_ATOMIC_NONBLOCK | DRM_MODE_PAGE_FLIP_EVENT, data);
```

Comment:
This flag appears to be required to send HDR metadata, otherwise I get `EINVAL` when committing.